README.md

```diff
@@ -227,7 +220,7 @@ To run the tests continuously for active development, run
 
 To run a specific test file, run
 
-    $ npm run test-file -- path/to/test/file
+    $ npm run test-file test/unit/SearchResultsPage.test.js
```

Comment:
this one and the one from above were changes that came up because i was supposed to cut from development and i cut from production, so i merged my changes back into development. i tried to suss out what needed to stay. i might just redo this again from the right branch

Reply:
@WormJim do you think i should redo it?